diff --git a/node_modules/matter-js/build/matter.dev.js b/node_modules/matter-js/build/matter.dev.js
index d006ee8..cd1eca3 100644
--- a/node_modules/matter-js/build/matter.dev.js
+++ b/node_modules/matter-js/build/matter.dev.js
@@ -8600,7 +8600,7 @@ var Mouse = __webpack_require__(14);
     if (typeof window !== 'undefined') {
         _requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame
                                       || window.mozRequestAnimationFrame || window.msRequestAnimationFrame
-                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 60); };
+                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 12); };
 
         _cancelAnimationFrame = window.cancelAnimationFrame || window.mozCancelAnimationFrame
                                       || window.webkitCancelAnimationFrame || window.msCancelAnimationFrame;
@@ -10449,7 +10449,7 @@ var Common = __webpack_require__(0);
         _requestAnimationFrame = function(callback){
             _frameTimeout = setTimeout(function() {
                 callback(Common.now());
-            }, 1000 / 60);
+            }, 1000 / 12);
         };
 
         _cancelAnimationFrame = function() {
diff --git a/node_modules/matter-js/build/matter.js b/node_modules/matter-js/build/matter.js
index 533a452..ababd7f 100644
--- a/node_modules/matter-js/build/matter.js
+++ b/node_modules/matter-js/build/matter.js
@@ -8599,7 +8599,7 @@ var Mouse = __webpack_require__(14);
     if (typeof window !== 'undefined') {
         _requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame
                                       || window.mozRequestAnimationFrame || window.msRequestAnimationFrame
-                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 60); };
+                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 12); };
 
         _cancelAnimationFrame = window.cancelAnimationFrame || window.mozCancelAnimationFrame
                                       || window.webkitCancelAnimationFrame || window.msCancelAnimationFrame;
@@ -10448,7 +10448,7 @@ var Common = __webpack_require__(0);
         _requestAnimationFrame = function(callback){
             _frameTimeout = setTimeout(function() {
                 callback(Common.now());
-            }, 1000 / 60);
+            }, 1000 / 12);
         };
 
         _cancelAnimationFrame = function() {
diff --git a/node_modules/matter-js/src/body/Body.js b/node_modules/matter-js/src/body/Body.js
index 5e51075..042a7f0 100644
--- a/node_modules/matter-js/src/body/Body.js
+++ b/node_modules/matter-js/src/body/Body.js
@@ -96,7 +96,7 @@ var Axes = require('../geometry/Axes');
             area: 0,
             mass: 0,
             inertia: 0,
-            deltaTime: 1000 / 60,
+            deltaTime: 1000 / 12,
             _original: null
         };
 
@@ -735,7 +735,7 @@ var Axes = require('../geometry/Axes');
      * @param {number} [deltaTime=16.666]
      */
     Body.update = function(body, deltaTime) {
-        deltaTime = (typeof deltaTime !== 'undefined' ? deltaTime : (1000 / 60)) * body.timeScale;
+        deltaTime = (typeof deltaTime !== 'undefined' ? deltaTime : (1000 / 12)) * body.timeScale;
 
         var deltaTimeSquared = deltaTime * deltaTime,
             correction = Body._timeCorrection ? deltaTime / (body.deltaTime || deltaTime) : 1;
diff --git a/node_modules/matter-js/src/core/Runner.js b/node_modules/matter-js/src/core/Runner.js
index dac30f9..07fbf50 100644
--- a/node_modules/matter-js/src/core/Runner.js
+++ b/node_modules/matter-js/src/core/Runner.js
@@ -37,7 +37,7 @@ var Common = require('./Common');
         _requestAnimationFrame = function(callback){ 
             _frameTimeout = setTimeout(function() { 
                 callback(Common.now()); 
-            }, 1000 / 60);
+            }, 1000 / 12);
         };
 
         _cancelAnimationFrame = function() {
diff --git a/node_modules/matter-js/src/render/Render.js b/node_modules/matter-js/src/render/Render.js
index 7c6d65c..60d5a14 100644
--- a/node_modules/matter-js/src/render/Render.js
+++ b/node_modules/matter-js/src/render/Render.js
@@ -26,7 +26,7 @@ var Mouse = require('../core/Mouse');
     if (typeof window !== 'undefined') {
         _requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame
                                       || window.mozRequestAnimationFrame || window.msRequestAnimationFrame
-                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 60); };
+                                      || function(callback){ window.setTimeout(function() { callback(Common.now()); }, 1000 / 12); };
 
         _cancelAnimationFrame = window.cancelAnimationFrame || window.mozCancelAnimationFrame
                                       || window.webkitCancelAnimationFrame || window.msCancelAnimationFrame;
