diff --git a/node_modules/matter-js/src/body/Body.js b/node_modules/matter-js/src/body/Body.js
index 5e51075..042a7f0 100644
--- a/node_modules/matter-js/src/body/Body.js
+++ b/node_modules/matter-js/src/body/Body.js
@@ -96,7 +96,7 @@ var Axes = require('../geometry/Axes');
             area: 0,
             mass: 0,
             inertia: 0,
-            deltaTime: 1000 / 60,
+            deltaTime: 1000 / 12,
             _original: null
         };
 
@@ -735,7 +735,7 @@ var Axes = require('../geometry/Axes');
      * @param {number} [deltaTime=16.666]
      */
     Body.update = function(body, deltaTime) {
-        deltaTime = (typeof deltaTime !== 'undefined' ? deltaTime : (1000 / 60)) * body.timeScale;
+        deltaTime = (typeof deltaTime !== 'undefined' ? deltaTime : (1000 / 12)) * body.timeScale;
 
         var deltaTimeSquared = deltaTime * deltaTime,
             correction = Body._timeCorrection ? deltaTime / (body.deltaTime || deltaTime) : 1;
diff --git a/node_modules/matter-js/src/core/Runner.js b/node_modules/matter-js/src/core/Runner.js
index dac30f9..07fbf50 100644
--- a/node_modules/matter-js/src/core/Runner.js
+++ b/node_modules/matter-js/src/core/Runner.js
@@ -37,7 +37,7 @@ var Common = require('./Common');
         _requestAnimationFrame = function(callback){ 
             _frameTimeout = setTimeout(function() { 
                 callback(Common.now()); 
-            }, 1000 / 60);
+            }, 1000 / 12);
         };
 
         _cancelAnimationFrame = function() {
